### ISSUE TITLE
How to skip duel on Linux?

### ISSUE THREAD

**Issue by foote-darrell** [jorio/BillyFrontier#6]
The ReadMe says we can skip the dual. What is the `⊞`key? Is this referring to the Windows key? The equivalent keystroke on Linux `Super`+`F10` does not seem to manifest this cheat.

**Comment by jorio** [comment]
Yes, it's the Windows key, aka "super" or "meta". If Win+F10 doesn't work for you, something in your desktop environment might be intercepting this key chord.

**Comment by foote-darrell** [comment]
Now how can I modify it to be a different key? The Windows/Super key
usually interacts with the desktop, so it is not very practical for
gameplay.

**Comment by jorio** [comment]
https://github.com/jorio/BillyFrontier/blob/d6abd2743c72a72ef7399d9eb6d9191889eb5eb3/Source/System/SDLInput.c#L628-L632

**Comment by foote-darrell** [comment]
Do `LGUI` and `RGUI` stand for the Super key? Where is the actual macro defined? searching for `SDL_SCANCODE_LGUI` doesn't show any other matches in the repository.

**Comment by jorio** [comment]
Search beyond the repo ;)
https://github.com/libsdl-org/SDL/blob/SDL2/include/SDL_scancode.h#L331
